plugins/extraction/extract_env.ml

```diff
@@ -722,7 +722,11 @@ let structure_for_compute env sg c =
 
 let compile f =
   try
-    let args = ["ocamlc";"-I";Filename.dirname f;"-c";f^"i";f] in
+    let args = ["ocamlc";
+                "-package";"zarith";
```

Comment:
Should this be `zarith` or `coq.extraction.runtime`?

Reply:
I think for now `zarith` should be fine. The whole point of using `zarith` is not to provide a special runtime for extraction. Also, Coq may require zarith soon so that could be fine. Now, it could fail if the user doesn't have it.